rollup.config.js

```diff
@@ -13,8 +13,9 @@ export default {
     babel({
       babelHelpers: 'bundled',
       exclude: 'node_modules/**',
+      extensions: ['.js', '.ts'],
     }),
-    nodeResolve(),
+    nodeResolve({ extensions: ['.js', '.ts'] }),
```

Comment:
No `.tsx` because none of the source modules in this project would use it, correct?

Reply:
Yes, exactly. We'll need to add this if we add Preact-based UI in future.